### PR TITLE
add Whats app integration

### PR DESCRIPTION
**Pull request by scopsy**

### CODE REVIEW

File: channels-and-providers/sms/twilio.mdx [hunk 1]
```diff
@@ -36,4 +36,21 @@ Creating a Twilio integration with Novu is quite simple. Just follow these steps
 - Go to your [Console](https://console.twilio.com/) on Twilio and access the Account Info section at the bottom of the page
 - Enter your `Account SID`, `Auth Token` and `Twilio Phone Number`.
 - Click on the **Save** button.
-- You should now be able to send SMS notifications using **Twilio** in Novu.
\ No newline at end of file
+- You should now be able to send SMS notifications using **Twilio** in Novu.
+
+## Sending an WhatsApp message with Twillio
```
Comment (SokratisVidros) [inline channels-and-providers/sms/twilio.mdx:41]: 💯

File: channels-and-providers/chat/whats-app.mdx [hunk 2]
```diff
@@ -1,19 +1,106 @@
...
-      subscriberId: '<UNIQUE_SUBSCRIBER_IDENTIFIER>',
-      phone: 'whatsapp:555-4242`
-    },
+To integrate WhatsApp Business with Novu, You will have to create a facebook developer app and obtain the necessary credentials.
+
+### Step 1: Create a Facebook Developer App
+
+Visit the [Facebook Developer Portal](https://developers.facebook.com/apps/) and create a new app.
+
+Select "Other" for "What do you want your app to do?" and select "Business" for "Select an app type".
```
Comment (SokratisVidros) [inline channels-and-providers/chat/whats-app.mdx:15]: Some screenshots will help.

File: channels-and-providers/chat/whats-app.mdx [hunk 3]
```diff
@@ -1,19 +1,106 @@
...
+
+On the App Setup page, click on "Set Up" under the "WhatsApp" product. You will need to create or add a Facebook Business Account to your app.
+
+### Step 3: Send a Sandbox Message
+
+Copy the following pieces and paste them in the Novu WhatsApp Business integration settings:
+- Temporary Access Token - Access API token Field
+- Phone Number ID - Phone Number Identification Field
+
+<Warning>
```
Comment (SokratisVidros) [inline channels-and-providers/chat/whats-app.mdx:27]: Maybe move the warning above the two bullets.

**Review by SokratisVidros — APPROVED** [review]

File: channels-and-providers/sms/twilio.mdx [hunk 4]
```diff
@@ -36,4 +36,21 @@ Creating a Twilio integration with Novu is quite simple. Just follow these steps
...
+
+To send a WhatsApps message with Twillio integration prefix the phone number of the subscriber with `whatsapp:` as shown below.
+
+```ts
+await novu.trigger('<WORKFLOW_TRIGGER_IDENTIFIER>',
+  {
+    to: {
+      subscriberId: '<UNIQUE_SUBSCRIBER_IDENTIFIER>',
+      phone: 'whatsapp:555-4242`
```
Comment (jainpawan21) [inline channels-and-providers/sms/twilio.mdx:50]:
```suggestion
      phone: 'whatsapp:555-4242'
```

**Pull request merged**